rop.spec

```diff
@@ -0,0 +1,56 @@
+Name:      rop
+Version:   0.0.1
+Release:   0%{?dist}
+Summary:   Install Foreman using containers
+
+License:   GPL-2-only
+URL:       https://github.com/theforeman/foreman-quadlet
+Source:    https://github.com/theforeman/foreman-quadlet/releases/download/%{version}/%{name}-%{version}.tar.gz
+
+BuildArch: noarch
+Requires:  ansible-collection-ansible-posix
+Requires:  ansible-collection-community-crypto
+Requires:  ansible-collection-community-general
+Requires:  ansible-collection-community-postgresql
+Requires:  ansible-collection-containers-podman >= 1.14.0
+Requires:  ansible-collection-theforeman-foreman
+Requires:  ansible-collection-theforeman-operations
```

Comment:
This dependency can be dropped with https://github.com/theforeman/foreman-quadlet/pull/155 which I think should go in first.

Reply:
This can be removed now

Reply:
```suggestion
```

please :)